src/index.ts

```diff
@@ -0,0 +1,134 @@
+import escape from 'xml-escape';
+import * as Cypress from 'cypress';
+import BeforeRunDetails = Cypress.BeforeRunDetails;
+import PluginConfigOptions = Cypress.PluginConfigOptions;
+import PluginEvents = Cypress.PluginEvents;
+import Spec = Cypress.Spec;
+import { ConfigOption, JUnitTestSuite } from './type';
+import Reporter, { TestCase, TestSuite } from './reporter';
+
+let reporter: Reporter;
+
+const onBeforeRun = (details: BeforeRunDetails) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  reporter.junitSuite.name = `Cypress Test - ${(details.config as any)?.projectName} ${details.browser?.displayName}`;
+};
+
+const onAfterSpec = (spec: Spec, results: CypressCommandLine.RunResult) => {
+  const testsuite = new TestSuite();
+  let errCount = 0;
+  results.tests.forEach((test) => {
+    const testcase = new TestCase(
+      test.title.join(' '),
+      test.title.length ? test.title[test.title.length - 1] : '',
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      msToSec((test as any).duration),
```

Comment:
I don't think this type assertion is necessary. `test.duration` is a known property.

Reply:
Oops. Removed.